### ISSUE TITLE
DINO Training with Swin-small 

### ISSUE THREAD

**Issue by YANGSHUAI2025** [IDEA-Research/detrex#334]
Dear author,
Hello.
I am now training dino, the swin-s chosen by the backbone. My configuration is the same as yours, 4gpus, but my batch_size is halved to 8, so the initial learning rate is halved, but the training results are all 0.
"d2.checkpoint.c2_model_loading WARNING: Shape of norm.weight in checkpoint is torch.Size([768]), while shape of necks.norm.weight in model is torch.Size([256]) "
"d2.checkpoint.c2_model_loading WARNING: Shape of norm.weight in checkpoint is torch.Size([768]), while shape of transformer.decoder.norm.weight in model is torch.Size([256])"I downloaded from techches website weight directly, is this why?
Please don't hesitate to enlighten me!

**Comment by rentainhe** [comment]
Hello, would u like to provide more info about your training config:
- the training config is the same as https://github.com/IDEA-Research/detrex/blob/main/projects/dino/configs/dino-swin/dino_swin_tiny_224_4scale_12ep.py or not?
- the pretrained swin checkpoint you use

And I think you don't have to half the batch_size and learning rate, you can use gradient_checkpoint to lower the gpu memory usage and keep the batch_size the same for training.

**Comment by YANGSHUAI2025** [comment]
Have you encountered similar problems when loading swin-s weights
<img width="1117" alt="微信图片_20240111174114" src="https://github.com/IDEA-Research/detrex/assets/124790061/9391c2b4-c03c-4a79-a23d-8a7840d80883">

**Comment by YANGSHUAI2025** [comment]
@rentainhe excuse me，Have you encountered similar problems when loading swin-s weights
<img width="1117" alt="微信图片_20240111174114" src="https://github.com/IDEA-Research/detrex/assets/124790061/8194d235-8a0d-484d-a373-5458c16045b6">
